backend/src/database/repositories/organizationRepository.ts

```diff
@@ -1295,7 +1295,14 @@ class OrganizationRepository {
     })
 
     if (!organization) {
-      return null
+      organization = await options.database.organization.create(
+        {
+          name: domain,
```

Comment:
We don't have a name field for organizations anymore, isn't this throwing an error? Names are stored in identities array instead now